src/components/crossDevice/CrossDeviceLink/style.css

```diff
@@ -17,14 +18,15 @@
 
 .numberInputSection {
   height: 48px;
-  width: 432px;
+  width: 100%;
   float: left;
   margin-top: 4px;
 }
 
 .inputContainer {
+  @base-width: 320;
   height: 100%;
-  width: 320px;
+  width: 100%*(@base-width/@parent-width);
```

Comment:
it's a repeated calculation, so it can be stored in a variable

Reply:
why do we need to do the calculation? `@base-width` and `@parent-width` are arbitrary values couldn't we just hard code the percentages?

Reply:
Initially I hard coded the percentage but in order to return the right value on big screens I had to use a lot of decimals which I don't like, therefore I'd rather have a reusable class that allows us to calculate a width of the child component based on the parent width.